src/index.ts

```diff
@@ -27,7 +27,7 @@ export {
   NgbPanelContent
 } from './accordion/accordion.module';
 export {NgbAlertModule, NgbAlertConfig, NgbAlert} from './alert/alert.module';
-export {NgbButtonsModule, NgbRadioGroup} from './buttons/radio.module';
+export {NgbButtonsModule, NgbRadioGroup} from './buttons/buttons.module';
```

Comment:
This is so people can query for it. We could add the new checkbox button here as well.